src/components/PaymentInformationForm/index.js

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react";
+import {
+  Main,
+  Choices,
+  Option,
+  Accommodation,
+} from "../PaymentInformationForm/PaymentWrapper";
+import AccomodationFinishMessage from "./AccomodationFinishMessage";
+
+export default function PaymentInformationForm() {
+  const [isSelected, setIsSelected] = useState({
+    isOnline: null,
+    price: null,
+    hashotelReservation: null,
+    hotelPrice: null,
+  });
+  const [isPresential, setIsPresential] = useState(false);
+  const [isOnlineOption, setIsOnlineOption] = useState(false);
+  const [hasHotel, setHasHotel] = useState(false);
+  const [hasntHotel, setHasntHotel] = useState(false);
+
+  function handleModalityChoice(isOnline, price) {
+    setIsSelected({
+      isOnline,
+      price,
+      hashotelReservation: null,
+      hotelPrice: null,
+    });
+    if (isOnline === false) {
+      setIsPresential(true);
+      setIsOnlineOption(false);
+      setHasntHotel(false);
+      setHasHotel(false);
+    } else {
+      setIsPresential(false);
+      setHasntHotel(false);
+      setHasHotel(false);
+      setIsOnlineOption(true);
+    }
+  }
+
+  function handleHospitalityChoice(choice, hotelPrice) {
+    if (choice === true) {
```

Comment:
Se choice é um booleano, estão a semântica poderia ser melhor. Pode começar a palavra com 'is' ou 'has', como vocês fizeram nas outras variáveis.

Reply:
Vou verificar!

Reply:
ok